### PR TITLE
Update examples

### LINKED ISSUE #512: Make Example equilibria consistent with their input files
some of our example equilibria are not consistent with a right-handed coordinate system (SOLOVEV, DSHAPE are two), because they were made with an old input file before we changed them to be right handed (for example the SOLOVEV input file is positive iota but SOLOVEV_output.h5 has a negative iota)

### LINKED ISSUE #677: Check NCSX input and output file for consistency
Force error is worse now when running input file as it is (but inverting iota helps somehow?) compared to the ncsx_output.h5

### PR DESCRIPTION
**Pull request by f0uriest**
- Removes old NCSX example with fixed iota (since it was a crappy fit)
- Renames QAS to NCSX (this is the correct one with fixed current)
- Updates ARIES-CS to be the actual correct configuration (what we have now seems to be a scaled down version)
- Adds example for HSX
- All example outputs have been regenerated on current master

Resolves #677 
Resolves #512

### CODE REVIEW

**Comment by rahulgaur104** [comment]
Can you also update the relevant files in tests/inputs? I couldn't re-run either of the LandremanPaul2022 equilibria.

**Comment by f0uriest** [comment]
> Can you also update the relevant files in tests/inputs? I couldn't re-run either of the LandremanPaul2022 equilibria.

Done. Also added all the examples you re-ran with the latest version.
Can you add the script you made to `desc/examples` so we have it for future use?

**Comment by rahulgaur104** [comment]
Done.

**Comment by rahulgaur104** [comment]
I think Unit tests / unit_tests (1, 3.8) (pull_request) fail because there is something wrong with the LadremanPaul2022 equilibria.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 =========
 
+- Updates `desc.examples`:
+    * `NCSX` now has a fixed current profile. Previously it used a fixed iota based on a
+    fit, but this was somewhat inaccurate.
+    * `QAS` has been removed as it is now redundant with `NCSX`.
+    * `ARIES-CS` has been scaled to the correct size and field strength.
+    * `WISTELL-A` is now a true vacuum solution, previously it approximated the vacuum
+    solution with fixed rotational transform.
+    * Flips sign of iota for `W7-X` and `ATF` to account for positive jacobian.
```
Comment (dpanici) [inline CHANGELOG.md:11]: why do we do this?

File: desc/examples/WISTELL-A [hunk 2]
```diff
@@ -8,44 +7,21 @@ NFP = 4
...
-# magnetic axis initial guess
-n:   0	R0 =   2.01244000E+00	Z0 =   0.00000000E+00
-n:   1	R0 =   4.05567000E-01	Z0 =   0.00000000E+00
-n:   2	R0 =   2.31526000E-02	Z0 =   0.00000000E+00
-n:   3	R0 =   4.72049000E-03	Z0 =   0.00000000E+00
-n:  -1	R0 =   0.00000000E+00	Z0 =  -2.96193000E-01
-n:  -2	R0 =   0.00000000E+00	Z0 =  -2.18210000E-02
-n:  -3	R0 =   0.00000000E+00	Z0 =  -2.54949000E-03
+objective         = vacuum
+spectral_indexing = ansi
```
Comment (dpanici) [inline desc/examples/WISTELL-A:24]: we dont include p or i or c bc it defaults to p and c of 0

**Review by dpanici — APPROVED** [review]

**Review by dpanici — DISMISSED** [review]

File: desc/examples/WISTELL-A (hunk 2 above)
Comment (f0uriest) [reply desc/examples/WISTELL-A:24]: yeah vacuum doesn't need any profiles

File: CHANGELOG.md (hunk 1 above)
Comment (f0uriest) [reply CHANGELOG.md:11]: previously I had flipped the sign of the boundary modes to give a positive jacobian but forgot to flip iota to match, this just fixes that.

**Pull request merged**